### PR TITLE
[WIP] Edit History Screen

### PR DESCRIPTION
**Pull request by aliseyalvi**
### What does this PR?
This PR adds edit history feature for  post which is available in ecency web app.
This is Work in Progress.

### Issue number
#2257 

### Screenshots/Video
image

### Update: 10 May 2022

https://user-images.githubusercontent.com/6298342/167702064-2e83a3c1-aa17-4261-a0cf-410b837d2d41.mov

### CODE REVIEW

File: src/screens/editHistoryScreen/editHistoryScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+      <View style={styles.mainContainer}>
+        {editHistory.length > 0 && _renderVersionsList()}
+        {editHistory.length > 0 && _renderBody()}
+      </View>
+    </Fragment>
+  );
+};
+
+export default EditHistoryScreen;
+
```
Comment (noumantahir) [inline src/screens/editHistoryScreen/editHistoryScreen.tsx:231]: Add some comment to mark them as webView styles for easy differentiation

File: src/providers/ecency/ecency.types.ts [hunk 2]
```diff
@@ -42,3 +42,11 @@ export interface UserPoint {
...
    [key:string]:QuoteItem
 }
+
+export interface CommentHistoryItem {
+    body: string;
+    tags: [string];
+    title: string;
+    timestamp:string;
+    v: number;
+}
```
Comment (noumantahir) [inline src/providers/ecency/ecency.types.ts:52]: add end of line

**Review by noumantahir — CHANGES_REQUESTED** [review]
I was never able to load the package because of some error related to $primaryBlack, I have added some code reviews however... will test again upon PR update.

Screenshot 2022-05-10 at 7 11 27 PM

**Review by noumantahir — APPROVED** [review]
@feruzm looking forward to you feedback, I have added updated demo video in PR description.

lgtm.

**Review by feruzm — APPROVED** [review]
lgtm 👍 lets do some device tests

**Comment by feruzm** [comment]
@noumantahir and @aliseyalvi we have highlight package using in Search page to highlight searched text, same can be used for highlighting modifications/diffs, something to consider in future.

**Pull request merged**